subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -53,6 +58,7 @@ static u8_t payload_buf[CONFIG_AWS_FOTA_PAYLOAD_SIZE];
 static u8_t hostname[CONFIG_AWS_FOTA_HOSTNAME_MAX_LEN];
 static u8_t file_path[CONFIG_AWS_FOTA_FILE_PATH_MAX_LEN];
 static u8_t job_id[AWS_JOBS_JOB_ID_MAX_LEN];
+static u8_t schema[8];
```

Comment:
Do I understand correctly it's up to `sizeof("https")`? Can it be any larger?

Reply:
It was an arbitrary value of the longest scheme name that should be allowed.

I'll try to look through the specification to see if there is actually a limit on how long the scheme can be but I don't think it will be very useful since I think we'll probably limit what we support to something like `http`,`https`,`mqtt`,`coap`,`tcp` and `udp`.  

And as for now we only support `http` and `https` so maybe limit it `sizeof("https")` is a good idea.